downloads/index.md

```diff
@@ -1,19 +1,37 @@
 # Download Julia
 
-~~~
-<a class="github-button" href="https://github.com/JuliaLang/julia" data-size="large" data-show-count="true" aria-label="Star JuliaLang/julia on GitHub">Star</a>
-~~~
-
 Please star us [on GitHub](https://github.com/JuliaLang/julia). If you use Julia in your research, please [cite us](https://julialang.org/research/). If possible, do consider [sponsoring](https://github.com/sponsors/julialang) us.
 
+
+## Juliaup: The recommended Julia installation manager
+
+To install Julia it is recommended to use the `juliaup` installation manager, which will automatically install
+and manage keeping julia up to date.
+
+Installation on Windows is either via the Windows store [here](https://www.microsoft.com/store/apps/9NJNWW8PVKMN)
+or via the command prompt:
+```
+winget install julia -s msstore
+```
+
+On Linux and MacOS:
+```
+curl -fsSL https://install.julialang.org | sh
+```
+
+Once installed `julia` and `juliaup` will be available via the command line interface.
+
+To install different versions see `juliaup --help` or the documentation at [github.com/JuliaLang/juliaup](https://github.com/JuliaLang/juliaup).
```

Comment:
IMO it would be good to have a suggestion here, like

```sh
juliaup add 1.9
juliaup default 1.9
```

So folks can get to a working Julia session without visiting more documentation if they don't need anything fancy. (The hardcoded version could be updated each release, as this page is already updated each release).

Reply:
Doesn't juliaup install latest automatically?

Reply:
Yes, you automatically get the latest version.

Reply:
ah ok!